registry/handlers/app.go

```diff
@@ -640,6 +640,7 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 		}
 
 		context := app.context(w, r)
+		defer defaultContextManager.release(context)
```

Comment:
@vikstrous Yea, unfortunately, based on what I discovered in the other patch, this addition to `net/http` _kind of_ backwards incompatible. Although, I'm not complaining, as the result is much better than it was.